pkg/skaffold/deploy/helm.go

```diff
@@ -70,18 +71,25 @@ func (h *HelmDeployer) Labels() map[string]string {
 	}
 }
 
-func (h *HelmDeployer) Deploy(ctx context.Context, out io.Writer, builds []build.Artifact, labellers []Labeller) error {
+func (h *HelmDeployer) Deploy(ctx context.Context, out io.Writer, builds []build.Artifact, labellers []Labeller) *Result {
 	var dRes []Artifact
 
 	event.DeployInProgress()
+	nsMap := map[string]bool{}
```

Comment:
I think `map[string]struct{}` is the recommended way to have a set in Go, since `struct{}` doesn't take up any memory!